### PR TITLE
Modal / Login / Signup components branch

### PR DESCRIPTION
**Pull request by edwindelbosque**
- [x] Add Modal component that includes both SignUpForm and SigninForm.
- [x] Add methods to make server requests for both SignUpForm creating a user and SigningForm getting access to a user.
- [x] ~~Add~~  **Remove** linter script to .json file
- [x] Style AccessModal
- [x] Style SignUpForm
- [x] Style SigninForm
- [x] Add routes and NavLinks to AccessModal to sign in and sign up components.
- [x] Change password type from 'text' to 'password' to hide passwords in inputs.
- [x] Add Links to SignIn and SignUp buttons when user submit form to get them back on movies page.
- [x] Add modal backdrop with opacity showing movie collage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "lint": "eslint src/",
```
Comment (vrandall66) [inline package.json]: Hey! I already added this on line 21, just remove one of them :)

**Comment by edwindelbosque** [comment]
[Style sign in page](https://trello.com/c/Ro3z6RAO/17-style-sign-in-page)

**Comment by edwindelbosque** [comment]
[Functionality to create a user - write POST method](https://trello.com/c/8x1i8cm9/5-functionality-to-create-a-user-write-post-method)

**Comment by ec-myers** [comment]
MVP will be form page - next iteration can be implementing react modal. Thanks Edwin!

**Pull request merged**